src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -30,6 +30,9 @@
 ## Loading files : https://stackoverflow.com/a/60687710
 import importlib.resources
 
+# Test Manipulation
+import textwrap
```

Comment:
Is this being used anywhere? Took a quick look and couldn't see it obviously used

Reply:
Removed in https://github.com/open-resources/problem_bank_scripts/pull/41/commits/72f545d8f7eb7c0426c37a3c199cdb774320774e